src/components/print/PrintDirective.js

```diff
@@ -174,6 +174,7 @@ goog.require('ga_urlutils_service');
         }
         canceller = $q.defer();
         $http.get(noCacheUrl, {
+          cache: false,
```

Comment:
Is it the default value, no?  There is a hack 2 line before for ie 9, should work for ie11

Reply:
With jquery 3 you don't need the hack anymore (untested) and may add the `cache: false`.